packages/admin/admin/src/dataGrid/muiGridFilterToGql.tsx

```diff
@@ -61,9 +61,13 @@ function convertValueByType(value: any, type?: string) {
 export function muiGridFilterToGql(columns: GridColDef[], filterModel?: GridFilterModel): { filter: GqlFilter; search?: string } {
     if (!filterModel) return { filter: {} };
     const filterItems = filterModel.items
-        .filter((filterItem) => filterItem.value !== undefined)
```

Comment:
if it is possible to have operators without value, this filter seems wrong, do we need it at all?

Reply:
Really? Then we have to either
- fix them
- or remove them

Reply:
is now fixed. was a problem for String-, Number- and DateTime-Filter.